### ISSUE TITLE
TitleWindow layout is not properly displayed

### ISSUE THREAD

**Issue by nihavend** [apache/royale-asjs#599]
For this case, there are two main issues.

1. TitleWindow cross image is not shown that closes window
2. VGroup is not displayed as in flex

Flex View : 

![image](https://user-images.githubusercontent.com/5983818/69900801-39d61480-1389-11ea-992f-9320cf5f7d57.png)

Royale View : 

![image](https://user-images.githubusercontent.com/5983818/69900798-29be3500-1389-11ea-98fa-8295aa9b5f43.png)

Test Case For Royale : 

```
<?xml version="1.0" encoding="utf-8"?>
<s:Application xmlns:fx="http://ns.adobe.com/mxml/2009"
			   xmlns:s="library://ns.apache.org/royale/spark"
			   xmlns:mx="library://ns.apache.org/royale/mx"
			   height="100%"
			   width="100%"
			   >	
	
	<fx:Metadata>
	</fx:Metadata>
	
	<fx:Script>
		<![CDATA[

			import mx.managers.PopUpManager;
			import mx.core.IUIComponent;
			
			private var fileViewWindow:FindBugFileViewPopup;
			
			public function closePopup():void {
				PopUpManager.removePopUp(fileViewWindow);
			}
			
			
			public function onClick():void {
				
				var fileTag:String = "infotag=myraConfig"
				fileViewWindow = PopUpManager.createPopUp(this as IUIComponent, FindBugFileViewPopup, true) as FindBugFileViewPopup;
				PopUpManager.centerPopUp(fileViewWindow);
				fileViewWindow.setFocus();
				
			}
			
		]]>
	</fx:Script>
	
	<fx:Declarations>		
	</fx:Declarations>	
	
	<mx:Panel title="Sample Code For Bug : 598"  paddingBottom="10" paddingTop="10" paddingLeft="10" paddingRight="10">
		<s:Button id="TwoNode" click="onClick()"  label="Show Popup"/>
		<s:Button id="close" click="closePopup()"  label="Close Popup"/>
	</mx:Panel>
	
</s:Application>
```

```
<?xml version="1.0"?>
<s:TitleWindow xmlns:fx="http://ns.adobe.com/mxml/2009"
			   xmlns:mx="library://ns.apache.org/royale/mx" 
			   xmlns:s="library://ns.apache.org/royale/spark"
			   creationComplete="titlewindow1_creationCompleteHandler(event)"
			   width="800" height="500">
	
	<s:layout> 
		<s:BasicLayout/>
	</s:layout>
	
	<fx:Script>
		<![CDATA[

			import mx.controls.Alert;
			import mx.events.FlexEvent;
			import mx.managers.PopUpManager;
			
			import spark.layouts.HorizontalLayout;
			
			private var _fileTag:String;
			public function get fileTag():String
			{
				return _fileTag;
			}
			public function set fileTag(fileTag:String):void
			{
				_fileTag = fileTag;
			}
			
			private var titlePrefix:String = "Log View Window";
			
			[Bindable]
			private var defEndOff:int = 5000;
			
			private var logViewCTX:String = "/logview";
			
			
			private function refreshService(event:Event):void {
				Alert.show("Refresh Button Clicke Event Received");
			}

			protected function titlewindow1_creationCompleteHandler(event:FlexEvent):void {
				title = titlePrefix;
			}
			
			
		]]>
	</fx:Script>
	
	<fx:Declarations>
	</fx:Declarations>
	
	<s:VGroup left="5" right="10" top="10" bottom="10" height="100%" width="100%">

		<mx:HBox id="firstRow" horizontalAlign="center" width="600">
			<s:FormItem label="Begin Offset" layout="{new HorizontalLayout()}" width="200" >
				<s:NumericStepper id="beginOff" stepSize="1" width="100" minimum="0" maximum="999999999" />
			</s:FormItem>
			<s:FormItem label="End Offset" layout="{new HorizontalLayout()}" width="200" >
				<s:NumericStepper id="endOff" stepSize="1" width="100" minimum="0" maximum="999999999" value="{defEndOff}"/>
			</s:FormItem>
			<s:FormItem width="200" >
				<s:Button label="Refresh" click="refreshService(event as Event)" />
			</s:FormItem>
		</mx:HBox>
		
		<mx:HRule width="100%" />
		
	</s:VGroup>
		
</s:TitleWindow>
```

**Comment by aharui** [comment]
This example is using FormItem in an expected way (no wrapping Form).  Spark Form/FormItem have not been emulated.  I'm hoping to find a quick way to emulate it.

Some other improvements were committed in 8af26f74e1e567ac47b936937a4072f2c2b19a53

Volunteers are welcome to make further improvements.

**Comment by nihavend** [comment]
I checked spark FormItem. I tought it was emulated. What is missing, do you mean the base classes are incomplete ?

**Comment by aharui** [comment]
It was emulated for compile-time, but I think it never really was made to run.  It looked like the TDF example was commented out.

**Comment by aharui** [comment]
FormItem is working better now.  There were a series of commits.  The latest is 5d7cc87

**Comment by nihavend** [comment]
1. TitleWindow cross image is not shown that closes window : Fixed 
2. VGroup is not displayed as in flex : It is working better, Fixed

I added handler for cross image and it works, closes window.
With the related commits, also the modal issue fixed.
There is a 2 simple problem but I am going to follow these cases with another issue, I am noting here just to inform.

1. ESC key is not working, that should close popup.
2. After pressing refresh button and than click on cross sign, the popup is closed but the modal state is not cleared I guess because the buttons on main page are not clickable any more.

Thanks @aharui I am closing the issue.
